frontend/src/lib/components/PropertiesTable/PropertiesTable.tsx

```diff
@@ -39,20 +38,24 @@ interface ValueDisplayType extends BasePropertyType {
 }
 
 function EditTextValueComponent({
-    value,
+    initialValue,
     onChange,
 }: {
-    value: any
-    onChange: (newValue: any, save: boolean) => void
+    initialValue: any
+    onChange: (newValue: any) => void
 }): JSX.Element {
+    const [value, setValue] = useState(initialValue)
+
     return (
-        <Input
-            defaultValue={value}
+        <LemonInput
```

Comment:
Oh, this actually should be an `EditableField` ideally. If it works well at this size, it will offer the smoothest experience

Reply:
It doesn't look to be a direct replacement. Even setting the buttons to be `xsmall` it still looks squished, and we need some alterations to the component to allow customisation of the rendered value in view mode

![test](https://github.com/PostHog/posthog/assets/6685876/e94bfe3e-1dc1-4b36-b4a0-1cf3be8dee9e)


Looks like `EditableField` has some other nuances in its API (see the `compactButton` type) so I might double back and fix that once the antd stuff has been removed

Reply:
Hmm, yeah, I guess that requires `xxsmall` 😄